vhdbuilder/packer/setupkata.sh

```diff
@@ -0,0 +1,45 @@
+while [ ! -f /opt/azure/containers/provision.complete ]
+do
+  echo "Waiting for provisioning to complete"
+  sleep 5
+done
+
+sleep 10
+
+cat /etc/containerd/config.toml | grep kata > /dev/null
+if [[ $? != 0 ]]; then
+  echo "kata config needs to be applied to containerd"
+  sed -i '/plugins."io.containerd.grpc.v1.cri".containerd]/a \\t  disable_snapshot_annotations = false' /etc/containerd/config.toml
```

Comment:
is it possible to make this correct by construction like we do in baker.go vs using unstructured text replace?

Reply:
Good suggestion! The current is to actually get rid of this file and use the IsKata() containerd configuration in AKS-RP instead. I'm drafting a design doc and will share it with you as soon as it's ready.